### DISCUSSION TITLE
Index images from multiple hard drives

### DISCUSSION THREAD

**Discussion by norton-chris** [photoprism/photoprism#2963] [General]
Is there a way to index images from multiple hard drives? I am currently using the OpenMediaVault 6 PhotoPrism plugin and I am able to successfully index one hard drive, but I have multiple hard drives on OMV that I would like to index as well.

**Comment by graciousgrey** [comment]
PhotoPrism itself allows you to mount different directories as subfolders of PhotoPrisms originals directory as described here:

https://docs.photoprism.app/getting-started/docker-compose/#photoprismoriginals

I am not sure if the OMV plugin provides an UI to do this.
